ingestors/golang.go

```diff
@@ -0,0 +1,70 @@
+package ingestors
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/buger/jsonparser"
+	"github.com/librariesio/depper/data"
+)
+
+const golangSchedule = "15 */6 * * *"
```

Comment:
This seems really infrequent... is it expensive to look more often?

Reply:
Does seem like waiting 6 hours to see a Module to show up in Libraries might be too long -- would 5 min be too often? (or maybe we could revisit all the ingestors later and tune them after running for a while)

Reply:
We had been doing it once daily :D 5 minutes sounds just dandy! And yes we should tune these at some point.

Reply:
Again sort of staving off making a smarter scheduler.